README.md

```diff
@@ -35,13 +35,13 @@ Multiple [demo applications](demos/src/main/java/com/datatorrent/demos) are prov
 Issue tracking
 --------------------
 
-[Malhar JIRA](https://malhar.atlassian.net/projects/MLHR) issue tracking system is used for this project.
-You can submit new issues and track the progress of existing issues at https://malhar.atlassian.net/projects/MLHR.
+[JIRA](https://issues.apache.org/jira/browse/APEXMALHAR) issue tracking system is used for this project.
+You can submit new issues and track the progress of existing issues at https://issues.apache.org/jira/browse/APEXMALHAR
 
-When working with JIRA to submit pull requests, please use [smart commits](https://confluence.atlassian.com/display/AOD/Processing+JIRA+issues+with+commit+messages) feature by specifying MLHR-XXXX in the commit messages.
-It helps us link commits with issues being tracked for easy reference.  And example commit might look like this:
+Please include the JIRA ticket number into the commit messages. It will add automatically add comments in JIRA and help link commits with issues being tracked for easy reference. 
```

Comment:
It will automatically add the commit message to the JIRA ticket(s) and help link the commit with the issue(s) being tracked for easy reference?